dashboard/src/components/partials/LabelsAnnotationsCell/LabelsAnnotationsCell.js

```diff
@@ -0,0 +1,140 @@
+import React from "react";
+import PropTypes from "prop-types";
+import gql from "graphql-tag";
+
+import { withStyles } from "@material-ui/core/styles";
+import CodeBlock from "/components/CodeBlock";
+import CodeHighlight from "/components/CodeHighlight/CodeHighlight";
+import CardContent from "@material-ui/core/CardContent";
+import Dictionary, {
+  DictionaryKey,
+  DictionaryValue,
+  DictionaryEntry,
+} from "/components/Dictionary";
+import Grid from "@material-ui/core/Grid";
+import Maybe from "/components/Maybe";
+import Label from "/components/partials/Label";
+
+const styles = () => ({
+  override: {
+    width: "25%",
+  },
+  fullWidth: {
+    width: "100%",
+  },
+});
+
+class LabelsAnnotationsCell extends React.PureComponent {
+  static propTypes = {
+    classes: PropTypes.object.isRequired,
+    entity: PropTypes.object,
+    check: PropTypes.object,
+  };
+
+  static defaultProps = {
+    entity: null,
+    check: null,
+  };
+
+  static fragments = {
+    check: gql`
+      fragment LabelsAnnotationsCell_check on CheckConfig {
```

Comment:
it would make the component simpler if we made the fragment only query the Metadata component, eg.

```graphql
fragment LabelsAnnotationsCell_metadata on Metadata {
  labels { key, val }
  annotations { key, val }
}
```

it would reduce the complexity and allow us to reuse it with any object that has annotations and labels.